src/plugins/console/server/lib/spec_definitions/json/generated/snapshot.cleanup_repository.json

```diff
@@ -10,6 +10,6 @@
     "patterns": [
       "_snapshot/{repository}/_cleanup"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/modules-snapshots.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/snapshot-restore/"
```

Comment:
`_cleanup` is not referenced in this documentation. Keep the link but add to `noDocumentation`